### PR TITLE
Display the number of variants per run on the Runs page

### LINKED ISSUE #389: Show the number of variants in a run on the /runs page

### PR DESCRIPTION
**Pull request by danvk**
Fixes #389 

To update your own database, run:

```sql
ALTER TABLE vcfs ADD COLUMN genotype_count BIGINT;
```

Then run this Python script (after sourcing `ENV.sh`) to populate the new field:

```python
import cycledash.runs
from workers.shared import initialize_database, update_vcf_count, DATABASE_URI

engine, connection, metadata = initialize_database(DATABASE_URI)

vcfs, last_comments, completions = cycledash.runs.get_runs()
for vcf_id in (v['id'] for v in vcfs):
    update_vcf_count(metadata, connection, vcf_id)
```

[Review on Reviewable](https://reviewable.io/reviews/hammerlab/cycledash/499)

### CODE REVIEW

File: cycledash/static/js/runs/components/RunsPage.js [hunk 1]
```diff
@@ -180,6 +181,7 @@ var RunRow = React.createClass({
         <td className='caller-name'>{run.caller_name}</td>
         <td className='dataset'>{run.dataset_name}</td>
         <td className='date' title={run.created_at}>{moment(new Date(run.created_at)).format('YYYY-MM-DD')}</td>
+        <td className='num-variants' title={run.genotype_count}>{run.genotype_count}</td>
```
Comment (ihodes) [inline cycledash/static/js/runs/components/RunsPage.js]: Need to increase the `colspan` on `RunDescription` by 1

**Comment by ihodes** [comment]
LGTM % 1 comment

**Comment by danvk** [comment]
Will merge on green. Once all my day's changes are in, I'll migrate the schema on `hammerlab-dev3`.

**Comment by ihodes** [comment]
You need to regenerate your dpxdt screenshots first!

**Comment by danvk** [comment]
truth

**Pull request merged**